ndt7/spec/spec.go

```diff
@@ -22,6 +22,13 @@ const MaxMessageSize = 1 << 24
 // a good compromise between Go and JavaScript as seen in cloud based tests.
 const MaxScaledMessageSize = 1 << 20
 
+// DefaultWebsocketBufferSize is the read and write buffer sizes used when
+// creating a websocket connection. This size is independent of the websocket
+// message sizes defined above (which may be larger) and used to optimize read
+// and write operations. However, larger buffers will practically limit the
+// total number of concurrent connections possible.
+const DefaultWebsocketBufferSize = 1 << 20
```

Comment:
Would prefer a comment after this `// 1 MB` or maybe `// 1 MiB` if you are feeling pedantic.